app/editor/src/features/admin/products/ProductForm.tsx

```diff
@@ -57,9 +58,6 @@ export const ProductForm: React.FC = () => {
     } catch {}
   };
 
-  /** ensure admin products is synced with lookup products when an admin changes a product lookup */
-  useLookupSync(handleSubmit);
```

Comment:
Removed this as it's a bad implementation.  We can just update state where we update state already (see comments below).